what-is-chalk.mdx

```diff
@@ -18,215 +18,440 @@ import { PyDiffEditor } from '@/components/Editor'
 
-When your model needs to use features that are canonically stored in
-a high-latency data source (like a data warehouse), Chalk's Reverse
-ETL support makes it simple to bring those features online and serve
-them quickly.
+- define cache windows inline with your features
+- override staleness at query-time when fresh data is critical
+- pre-warm caches and backfill pipelines for performance
 
-Add a single line of code to an `offline` resolver, and Chalk constructs
-a managed reverse ETL pipeline for that data source:
+Add a caching policy with one line of code in our feature definition:
 
-```py
-@offline(offline_to_online_etl="5m")
-```
+<PyDiffEditor html={addCachingCode} />
 
-Now data from slow offline data sources is automatically available for
-low-latency online serving.
 
----
+### How do I deploy and query features with Chalk?
+
+Once we've defined our pipelines, deploy features to a branch in < 30s with:
 
-## Deploying and querying
+```shell
+chalk apply --branch=elvis
+```
+
+We can then query for features against this branch and perform QA workflows on our preview environment to make sure that our Chalk deployment performs as expected:
+
+```shell
+chalk query --branch=elvis --in user.id=47
+```
 
-Once you've defined your pipelines, you can rapidly deploy them to
-production with [Chalk's CLI](/cli/introduction):
+Once validated, promote the deployment to production,
+which is served at a unique preview URL.
 
-```bash
+```shell
 chalk apply
 ```
 
-This creates a deployment of your project, which is served at a unique
-preview URL. You can promote this deployment to production, or
-perform QA workflows on your preview environment to make sure that
-your Chalk deployment performs as expected.
+These features are now available for low-latency online inference and [offline training](https://docs.chalk.ai/docs/training-client).
+The ability to re-use the same features in both contexts dramatically shortens development time and ensures that feature values from online and offline contexts match.
+
+## Why Chalk?
+
+Chalk is the first platform to unify feature engineering, LLM orchestration, and real-time inference under a single abstraction: programmable features.
+
+At the core, Chalk retrieves and computes features dynamically at inference time with an execution engine called Velox.
+We maintain a fork that’s been heavily optimized for low-latency (< 10ms).
+
+This enables teams to:
+
+- consolidate training and serving into a single version controlled environment
+- reduce time-to-production for new models from months to days
+- build advanced AI systems (RAG, RecSys, fraud, etc.) without building custom infrastructure
+
+You write Python. Chalk handles orchestrating, caching, and serving features at scale, resulting in faster iterations, fewer bugs, and the freedom to focus on building models—not plumbing.
```

Comment:
in general, there's some switching between "you", "we", and more imperative voice "Once validated, deploy ...". I'm not sure exactly which one we should use, but I think we could be more consistent.